### PR TITLE
Better staff who verb window

### PR DESCRIPTION
**Pull request by DreamySkrell**
image
image

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!review

File: code/game/verbs/who.dm [hunk 1]
```diff
@@ -9,44 +9,77 @@
...
 			if(is_special_character(client.mob))
-				msg += " - <b><span class='warning'>Antagonist</span></b>"
-			msg += " (<A HREF='?_src_=holder;adminmoreinfo=\ref[client.mob]'>?</A>)"
-			msg += "<br>"
+				msg += "<td><span class='warning'>Antagonist</span></b></td>"
+			else
+				msg += "<td></td>"
+
+			// more info button
+			msg += "<td><A HREF='?_src_=holder;adminmoreinfo=\ref[client.mob]'>?</A></td>"
```
Comment (FluffyGhoster) [inline code/game/verbs/who.dm:78]: REF() over \ref

File: code/game/verbs/who.dm [hunk 2]
```diff
@@ -9,44 +9,77 @@
...
+				msg += "<td>[round(client.avgping, 1)]ms</td>"
+			else
+				msg += "<td></td>"
 
+			// antag
 			if(is_special_character(client.mob))
-				msg += " - <b><span class='warning'>Antagonist</span></b>"
-			msg += " (<A HREF='?_src_=holder;adminmoreinfo=\ref[client.mob]'>?</A>)"
-			msg += "<br>"
+				msg += "<td><span class='warning'>Antagonist</span></b></td>"
```
Comment (FluffyGhoster) [inline code/game/verbs/who.dm:73]: SPAN_WARNING

**Review by FluffyGhoster — CHANGES_REQUESTED** [review]

File: code/game/verbs/who.dm (hunk 2 above)
Comment (DreamySkrell) [reply code/game/verbs/who.dm:73]: fixed

File: code/game/verbs/who.dm (hunk 1 above)
Comment (DreamySkrell) [reply code/game/verbs/who.dm]: fixed

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**